### PR TITLE
Webhooks - additional tests, fixes, refactoring

### PR DESCRIPTION
**Pull request by yromanyshyn**

### CODE REVIEW

File: backend/deepchecks_monitoring/schema_models/alert_webhook.py [hunk 1]
```diff
@@ -238,14 +238,14 @@ def prepare_request_parameters(self, alert: "Alert", settings: "Settings") -> t.
...
-        if self.kind == WebhookKind.STANDART:
+        if self.kind == WebhookKind.STANDARD:
             return {
                 "url": self.http_url,
                 "method": self.http_method,
                 "headers": self.http_headers,
                 "json": {
                     "alert_id": alert.id,
-                    "alert_name": f"models/{model.name}/monitors/{monitor.name}",
+                    "alert_name": f"models: {model.name} monitors: {monitor.name}",
```
Comment (matanper) [inline backend/deepchecks_monitoring/schema_models/alert_webhook.py:248]: maybe in singular form?
"model: x, monitor: y"

File: backend/deepchecks_monitoring/schema_models/alert_webhook.py [hunk 2]
```diff
@@ -270,18 +270,19 @@ def prepare_request_parameters(self, alert: "Alert", settings: "Settings") -> t.
...
                         "component": "deepchecks",
                         "group": additional_arguments.get("group") or "deepchecks",
                         "class": additional_arguments.get("class"),
                         "custom_details": {
-                            "deepchecks_alert_id": alert.id,
-                            "deepchecks_alert_start_time": str(alert.start_time),
-                            "deepchecks_alert_end_time": str(alert.end_time),
-                            "deepchecks_alert_failed_values": alert.failed_values,
+                            "alert_id": alert.id,
+                            "alert_name": f"models/{model.name}/monitors/{monitor.name}",
```
Comment (matanper) [inline backend/deepchecks_monitoring/schema_models/alert_webhook.py:282]: maybe alert_link?

File: backend/deepchecks_monitoring/schema_models/alert_webhook.py (hunk 1 above)
Comment (yromanyshyn) [reply backend/deepchecks_monitoring/schema_models/alert_webhook.py]: yeap, fixed

File: backend/deepchecks_monitoring/schema_models/alert_webhook.py (hunk 2 above)
Comment (yromanyshyn) [reply backend/deepchecks_monitoring/schema_models/alert_webhook.py]: no, it is not a link
it also should be `f"model: {model.name} monitor: {monitor.name}",`

**Review by matanper — APPROVED** [review]

**Pull request merged**